src/Analyzers/Core/CodeFixes/NamespaceSync/AbstractNamespaceSyncCodeFixProvider.MyCodeAction.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.CodeFixes.NamespaceSync
+{
+    internal abstract partial class AbstractNamespaceSyncCodeFixProvider
+    {
+        private sealed class MyCodeAction : CustomCodeActions.SolutionChangeAction
+        {
+            public MyCodeAction(string title, Func<CancellationToken, Task<Solution>> createChangedSolution)
+                : base(title, createChangedSolution)
```

Comment:
i'm surprised this didn't have an equivalence key.

Reply:
Not sure I follow. I thought EquivalenceKey was for if two fixes could be presented in the same lightbulb? I guess it could happen if someone did... 

```csharp
namespace A { class C1 { } } namespace B { class C2 { } }
```